safe/gui/tools/wizard_dialog.py

```diff
@@ -167,25 +169,26 @@
 step_kw_aggregation = 11
 step_kw_source = 12
 step_kw_title = 13
-step_fc_function_1 = 14
-step_fc_function_2 = 15
-step_fc_function_3 = 16
-step_fc_hazlayer_origin = 17
-step_fc_hazlayer_from_canvas = 18
-step_fc_hazlayer_from_browser = 19
-step_fc_explayer_origin = 20
-step_fc_explayer_from_canvas = 21
-step_fc_explayer_from_browser = 22
-step_fc_disjoint_layers = 23
-step_fc_agglayer_origin = 24
-step_fc_agglayer_from_canvas = 25
-step_fc_agglayer_from_browser = 26
-step_fc_agglayer_disjoint = 27
-step_fc_extent = 28
-step_fc_extent_disjoint = 29
-step_fc_params = 30
-step_fc_summary = 31
-step_fc_analysis = 32
+step_kw_summary = 14
```

Comment:
For when you have time - could we not get these numbers by querying the Qt object tree and so keep the code below a little more future proof?